### PR TITLE
Update deprecation warning for out of scope resolution

### PR DESCRIPTION
**Pull request by ljacomet**

### CODE REVIEW

**Comment by ljacomet** [comment]
@bot-gradle test RFM

File: subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/viewing_debugging_dependencies.adoc [hunk 1]
```diff
@@ -207,6 +207,7 @@ For example:
 * A task from one project directly resolves a configuration in another project.
 * A task specifies a configuration from another project as an input file collection.
 * A build script for a project resolves a configuration in another project during evaluation.
+* Project configurations are resolved from project settings
```
Comment (big-guy) [inline subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/viewing_debugging_dependencies.adoc:210]: what does "project settings" mean here?

**Review by big-guy — COMMENTED** [review]
the actual change looks OK, question about the docs

File: subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/viewing_debugging_dependencies.adoc (hunk 1 above)
Comment (ljacomet) [reply subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/viewing_debugging_dependencies.adoc:210]: `settings.gradle(.kts)`
How would you phrase that then?

**Pull request merged**